server/routes/screenshot/html.py

```diff
@@ -27,6 +31,48 @@
 bp = flask.Blueprint("screenshot", __name__, url_prefix='/screenshot')
 
 
+@bp.route('/')
+def list():
+  base_sha = request.args.get('base', '')
+  g = Github("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")
```

Comment:
Can you add a comment describing where this access token came from

Reply:
Store this token(and regenerated) in secret manager now. Added a comment on where this is created.